src/AbpTemplate.Core/Localization/SourceFiles/AbpTemplate-ja.xml

```diff
@@ -3,7 +3,7 @@
   <texts>
     <text name="HomePage" value="ホームページ" />
     <text name="About" value="我々について" />
-    <text name="WellcomeMessage" value="ようこそ AbpTemplate へ!" />
+    <text name="WellcomeMessage" value="ようこそ WebEdi へ!" />
```

Comment:
instead of **WebEdi** we should use **AbpTemplate** I guess.

Reply:
Haha whoops.